Document/0x06a-Platform-Overview.md

```diff
@@ -12,20 +12,21 @@ Like the Apple desktop operating system macOS (formerly OS X), iOS is based on D
 
 However, iOS apps run in a more restricted environment than their desktop counterparts do. iOS apps are isolated from each other at the file system level and are significantly limited in terms of system API access.
 
-To protect users from malicious applications, Apple restricts and controls access to the apps that are allowed to run on iOS devices. The Apple App store is the only official application distribution platform. There developers can offer their apps and consumers can buy, download, and install apps. This distribution style differs from Android, which supports several app stores and sideloading (installing an app on your iOS device without using the official App store).
+To protect users from malicious applications, Apple restricts and controls access to the apps that are allowed to run on iOS devices. Apple's App Store is the only official application distribution platform. There developers can offer their apps and consumers can buy, download, and install apps. This distribution style differs from Android, which supports several app stores and sideloading (installing an app on your iOS device without using the official App Store, 

```

Comment:
In android there are various ways to do this, and this is not often via usb, so this might be removed as well.

Reply:
In Android there are multiple ways to distribute apps from untrusted sources: ADB via USB, aDB via TCP/IP, email, web server, etc. In iOS, sideloading typically refers to the installation via USB, although there are other Enterprise app distribution methods.